_topic_map.yml

```diff
@@ -1678,6 +1680,9 @@ Topics:
 ### Brokers
 ###  - Name: Using the broker
 ###    File: serverless-using-brokers
+### HA
+### - Name: Configuring High Availability components
```

Comment:
Unless there's something special w/ it in serverless, "high availability" doesn't need to be capitalized. Here and in the heading of the assembly.